ansible/roles/elasticsearch/templates/create_es_snapshot.py.j2

```diff
@@ -18,16 +23,31 @@ def create_snapshot(client):
 
 def delete_old_snapshots(client):
     snaps = client.get('{{ es_repository_name }}', '_all')
-    three_days_ago = (datetime.utcnow() - timedelta(days=3)).isoformat()
-    old_snapshots = [s['snapshot'] for s in snaps['snapshots'] if s['start_time'] < three_days_ago]
+    number_of_days_ago = (datetime.utcnow() - timedelta(days={{elastcsearch_backup_days|default(3)}} )).isoformat()
```

Comment:
To change the days to keep elasticsearch backup of

Reply:
I think now that deletes will start to work we should reconsider how long we set this. ES backups are very lightweight since they only store each document once per version. Since we pretty much never delete documents and only a small percentage are ever changed (the majority are forms which are basically static), extra days do not increase storage costs very much. Maybe a month makes sense as a default, to give us a little more flexibility in restoring.
@javierwilson @NitigyaS

Reply:
I agree... and S3 is also very cheap. Also, Amazon has a policy that allows you to move old S3 files to Glacier, right @NitigyaS  ?  Perhaps older than 3-days but younger than 30-days should go to glacier?

@calellowitz this code allows us to set elastcsearch_backup_days in public vars, so we can set that there, and have 30 as a default in default common vars (i think it's groups vars all, unless it has been moved)

Reply:
@javierwilson 
I am not sure that glacier plan would work. The snapshots are not complete, they are incremental, so pushing them to glacier does not buy us much in terms of storage savings, but doubles our storage usage since everything is in glacier and s3. We might as well just leave them all in s3